teams/leadership-council.toml

```diff
@@ -0,0 +1,40 @@
+name = "leadership-council"
+
+[people]
+leads = []
+members = [
+    "carols10cents",
+    "eholk",
+    "ehuss",
+    "jackh726",
+    "jonathanpallant",
+    "khionu",
+    "Mark-Simulacrum",
+    "m-ou-se",
+    "rylev",
+]
+alumni = []
+
+[permissions]
+bors.rust.review = true
+bors.team.review = true
```

Comment:
The "team" bors review permissions is not really used (this repo doesn't use bors).